client-app/src/admin/AppModel.js

```diff
@@ -36,7 +36,12 @@ export class AppModel extends BaseAppModel {
             },
             {
                 name: 'wip',
-                path: '/wip'
+                path: '/wip',
+                children: [
+                    {name: 'simpleTreeMap', path: '/simpleTreeMap'},
+                    {name: 'gridTreeMap', path: '/gridTreeMap'},
+                    {name: 'splitTreeMap', path: '/splitTreeMap'}
```

Comment:
I think we can move these to the main Toolbox app at this point.  Looking for a succinct name for the Charts tab that allows them to live under there... Visualizations?  DataViz?  Leave as charts?